### ISSUE TITLE
Choice between Gaussian and Binary

### ISSUE THREAD

**Issue by pranavkr29** [KarenUllrich/binary-VAE#1]
Hey, 
I recently came across your code, and was trying to see if it ran('cause the official tensorflow version is giving some dataset autograph error). 

Your code works very well, but I'd like to ask where do you specify the choice between a Discrete and Gaussian distribution? in the experiment.py file, there is an if condition between the two, but nowhere a choice, and we run the code as "python3 experiment.py". And yet it complies perfectly. 

A secondary and less important question was that the generated images don't seem as binary(the images have a grayish quality to them as opposed to the training data we're plotting them against). So I printed the reconstructed output against the data, and yeah the recon array was decimal values from 0 to 1, not binary. How to fix that? By using a sigmoid activation in the final layer?

thanks in advance for your time
